src/main/java/com/redhat/devtools/intellij/quarkus/lsp/QuarkusLanguageClient.java

```diff
@@ -185,4 +190,21 @@ public CompletableFuture<CodeAction> resolveCodeAction(CodeAction unresolved) {
       return (CodeAction) PropertiesManagerForJava.getInstance().resolveCodeAction(unresolved, PsiUtilsLSImpl.getInstance(getProject()));
     });
   }
+
+  @Override
+  public CompletableFuture<JavaCursorContextResult> getJavaCursorContext(MicroProfileJavaCompletionParams params) {
```

Comment:
I think this method is not used in MP LS, let's keep like this, but I think we should remove it,see https://github.com/eclipse/lsp4mp/issues/408